### PR TITLE
Added uphold.com and principal.com icons

### PR DESCRIPTION
**Pull request by SudoEvan**

### CODE REVIEW

File: vectors/principal-financial/information.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "domain": "principal.com"
+  }
```
Comment (justindbaur) [inline vectors/principal-financial/information.json:3]: Braces don't line up.

File: vectors/principal-financial/principal-financial.svg [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      <stop offset="0.4" style="stop-color:rgb(33,143,207);stop-opacity:1"></stop>
+      <stop offset="0.5" style="stop-color:rgb(45,151,211);stop-opacity:1"></stop>
+      <stop offset="0.6" style="stop-color:rgb(46,161,213);stop-opacity:1"></stop>
+      <stop offset="0.7" style="stop-color:rgb(48,171,214);stop-opacity:1"></stop>
+      <stop offset="0.8" style="stop-color:rgb(50,180,215);stop-opacity:1"></stop>
+      <stop offset="0.9" style="stop-color:rgb(53,190,215);stop-opacity:1"></stop>
+      <stop offset="1" style="stop-color:rgb(55,199,216);stop-opacity:1"></stop>
+    </linearGradient>
+  </defs>
+</svg>
```
Comment (justindbaur) [inline vectors/principal-financial/principal-financial.svg:20]: Add newline at end of file

File: vectors/uphold/uphold.svg [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+</svg>
```
Comment (justindbaur) [inline vectors/uphold/uphold.svg:4]: Newline as well

**Review by justindbaur — COMMENTED** [review]
Just a few nitpicks.

File: vectors/principal-financial/information.json (hunk 1 above)
Comment (SudoEvan) [reply vectors/principal-financial/information.json]: Fixed in commit a7235cc

File: vectors/principal-financial/principal-financial.svg (hunk 2 above)
Comment (SudoEvan) [reply vectors/principal-financial/principal-financial.svg]: Also fixed in commit a7235cc

File: vectors/uphold/uphold.svg (hunk 3 above)
Comment (SudoEvan) [reply vectors/uphold/uphold.svg]: Also fixed in commit a7235cc, thanks!

**Pull request merged**